docs/architecture/porting-existing-aspnet-apps/signalr-differences.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Comparing SignalR and SignalR Core
+description: 
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Comparing SignalR and SignalR Core
```

Comment:
Let's refer to them as "ASP.NET SignalR" and "ASP.NET Core SignalR" to align with the docs.